client/app/_groups/groups.controller.js

```diff
@@ -198,13 +181,13 @@ const GroupsController = function GroupsController(
                 }
             });
         };
+
         $scope.updateGroup = function (group) {
-            group.permissions = $scope.permissions.filter(function (permission) {
-                return permission.value;
-            });
-            GroupService.updateGroup(group).then(function (rs) {
+            group.permissions = $scope.allAvaliablePermissions.filter((permission) => permission.value).map((permission) => permission.name);
```

Comment:
I've seen this line. Please, follow the DRY principle, if possible. And this line is too long